bugs/gh-207.txt

```diff
@@ -0,0 +1,309 @@
+=== Begin SRU Template ===
+[Impact]
+A force flag (-f) was added to command line args when creating a swap
+filesystem
+
+[Test Case]
+#!/bin/bash
+
+cat >vm_profile.yaml <<EOF
+config: {}
+description: Default LXD profile for VMs
+devices:
+  config:
+    source: cloud-init:config
+    type: disk
+  eth0:
+    name: eth0
+    network: lxdbr0
+    type: nic
+  root:
+    path: /
+    pool: default
+    type: disk
+name: vm
+EOF
+
+cat >test-case.yaml <<EOF
+#cloud-config
+ssh_authorized_keys:
+ - |
+  ssh-rsa <your ssh key>
+
+disk_setup:
+    /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_extra-disk:
+        table_type: gpt
+        layout: [[100, 82]]
+        overwrite: True
+fs_setup:
+    - device: /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_extra-disk
+      partition: 1
+      filesystem: swap
+      overwrite: True
+mounts:
+    - ["/dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_extra-disk", "none", "swap", "sw", "0", "0"]
+
+EOF
+
+SERIES_LIST='focal eoan bionic'  # Xenial was verified manually due to lxd-agent issues
+
+function wait_till_ready() {
+  name=$1
+  lxc exec $name -- /bin/bash -c ":"
+  while [ $? -ne 0 ]; do
+    lxc exec $name -- /bin/bash -c ":"
```

Comment:
Good point. I didn't do the sleep because when I first tried it it waited a second or so before failing. I see in the verification a wall of failures though, so a sleep is a good idea. :smiley: